Author: Petr Prochazka <petrprochy@gmail.com>
Descsription: [PATCH] SWT GTK 3.7.1 fixes
 SWT GTP Widget API was changed.

Index: azureus-4.7.0.0/org/eclipse/swt/widgets/Tree2.java
===================================================================
--- azureus-4.7.0.0.orig/org/eclipse/swt/widgets/Tree2.java	2011-10-26 00:03:50.116578204 +0200
+++ azureus-4.7.0.0/org/eclipse/swt/widgets/Tree2.java	2011-10-26 00:04:24.357859919 +0200
@@ -48,10 +48,10 @@
 		// skip check
 	}
 
-	// @see org.eclipse.swt.widgets.Tree#createHandle()
-	void createHandle() {
+	// @see org.eclipse.swt.widgets.Tree#createHandle(int index)
+	void createHandle(int index) {
 		// lucky for us, other platforms have a super.createHandle too
-		super.createHandle();
+		super.createHandle(index);
 		//if (explorerTheme) {
 		//	int bits2 = (int)/*64*/OS.SendMessage (handle, OS.TVM_GETEXTENDEDSTYLE, 0, 0);
 		//	bits2 &= ~OS.TVS_EX_FADEINOUTEXPANDOS;
